packages/atomic/cypress/utils/setupComponent.ts

```diff
@@ -1,9 +1,16 @@
-export function injectComponent(componentInCode: string) {
-  cy.get('atomic-search-interface').should('exist');
-  cy.document().then((document: any) => {
-    document.querySelector(
-      'atomic-search-interface'
-    ).innerHTML = componentInCode;
+const searchInterfaceTag = 'atomic-search-interface';
+export function injectComponent(
+  componentHtml: string,
+  executeFirstSearch = true
+) {
+  cy.document().then(async (document) => {
```

Comment:
Finally figured it out...
We had to "inject" the slotted component with the atomic-search-interface in order to be able to lauch a first search straight away. Not doing so would sometimes cause a race condition between the facet registration and the launch of the search. When both are injected together, they will always have the same "lifecycle"... I hope that's clear 😅

Reply:
awesome, thanks so much @ThibodeauJF